src/components/Status.js

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import {
+  SprkLink,
+  SprkIcon,
+  SprkTooltip
+} from '@sparkdesignsystem/spark-react';
+
+const Status = ({
+    statusName,
+    href,
+    additionalInfo
+}) => {
+    if (statusName === "ready") {
+        return (
+            <div>
+                <SprkIcon className="
+                    docs-status__Ready 
+                    sprk-c-Icon--filled
+                    sprk-c-Icon--l"
+                    iconName="check-mark-filled"
+                />
+                <SprkLink
+                    element="a"
+                    href={href}
+                    variant="simple"
+                    additionalClasses="sprk-u-Color--green docs-link__green"
+                >
+                    Ready
+                </SprkLink> 
+            </div>
+        );
+    } else if (statusName === "underReview") {
+        return (
+            <div>
+                <SprkIcon className="
+                    docs-status__UnderReview
+                    sprk-c-Icon--filled
+                    sprk-c-Icon--l"
+                    iconName="caution-filled"
+                />
+                <SprkLink
+                    element="a"
+                    href={href}
+                    variant="simple"
+                    additionalClasses="docs-status-text__Orange docs-link__yellow"
```

Comment:
something feels weird about why these have two colors. Is it orange or is it yellow? I'm noticing that `docs-link__yellow` has an orange underline. Why is that and why are they in separate classes?